src/components/datatable/datatable.scss

```diff
@@ -0,0 +1,78 @@
+.datatable {
```

Comment:
Using SAS is not allowed in this project for a reason; we provided guidelines on how it should be structured. When working on a collaborative project, we can't just add elements that are not pre-set within the software architecture. I cannot continue reviewing your pull request, since an important guideline was not followed